spec/openapi.yaml

```diff
@@ -258,13 +258,13 @@ paths:
                 externalValue: 'https://raw.githubusercontent.com/kartverket/SFKB-API/master/spec/examples/linestring/Mønelinje.json'
               singleArc:
                 description: |
-                  Eksempel på uthentet bue (WIP)
+                  Eksempel på uthentet bue
                   
                   Standarden GeoJSON har ingen egen type for bue. 
                   
-                  For at buer enkelt skal kunne leses av klienter, er derfor bue-geometrien konvertert til en linje med `is_arc=true` i `geometry_properties`.
+                  For at buer enkelt skal kunne leses av klienter, er derfor bue-geometrien konvertert til en linje med `is_circular=true` i `geometry_properties`.
                   Første, midtre og siste punkt beskriver buen dersom klienten vil implementere korrekt tegning av denne.
```

Comment:
Såvidt jeg forstår så trenger det ikke kun å være tre punkter her? Hva brukes de siste punktene til?

Reply:
De resterende punktene er buen generert som en "vanlig" linje - for å gjøre det enkelt å tegne opp buen med standard GeoJSON dersom man ikke ønsker å forholde seg til utvidelsen `geometry_properties`. Dette gjelder kun for opptegning/lesing av dataene - ved oppdatering er det de tre buepunktene (første, midtre og siste) som gjelder.

Har du et tekstforslag til hva som bør inn i dokumentasjonen for at dette skal være klart? :)

Reply:
Første avsnitt i svaret ditt her er perfekt etter linje 266